dev-requirements.txt

```diff
@@ -1,2 +1,3 @@
 flake8
 pytest-ckan
+-e git+https://github.com/okfn/okfn_iati.git@0.1.0#egg=okfn_iati
```

Comment:
This is a hard requirement for the extension itself, not only for devs.
This must be part of a `requirements.txt` file, not `dev-requirements.txt`
The  `dev-requirements.txt` fiel is for devs (e.g. for testing purposes)